### ISSUE TITLE
Develop ButtonList component

### ISSUE THREAD

**Issue by TheHadiAhmadi** [yesvelte/yesvelte#150]
https://preview.tabler.io/docs/buttons.html (List of Buttons)
<img width="571" alt="image" src="https://user-images.githubusercontent.com/42554876/221556554-8d2005fe-3c02-4b9d-b306-d9eb6f8c162f.png">

**Comment by pournasserian** [comment]
@TheHadiAhmadi Can we add a property to ButtonGroup for the above UI?

**Comment by TheHadiAhmadi** [comment]
@pournasserian Yes it is possible. something like compact=false (we had this feature about 4 - 5 months ago for ButtonGroup)

**Comment by pournasserian** [comment]
> @pournasserian Yes it is possible. something like compact=false (we had this feature about 4 - 5 months ago for ButtonGroup)

@TheHadiAhmadi 
I saw that: `compact` prop name is confusing. ButtonGroup is compact by default. I agree to have that feature, but what should be the prop name (like list:true/false)?

**Comment by TheHadiAhmadi** [comment]
ButtonGroup => .btn-list
ButtonGroup compact => .btn-group

ButtonGroup can be btn-list by default
